runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2713,6 +2713,7 @@ gcInitializeDefaults(J9JavaVM* vm)
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	/* after we parsed cmd line options, check if we can obey the request to run CS */
 	if (extensions->concurrentScavengerForced) {
+#if !defined(S390) && !defined(J9ZOS390)
```

Comment:
Would it be easier to read ?
#if !(defined(S390) || defined(J9ZOS390))

Reply:
changed to #if !(defined(S390) || defined(J9ZOS390))